appliances/FELDSAM/5A435F05-412D-4FC9-930A-0E6295760A78.yaml

```diff
@@ -0,0 +1,83 @@
+--- 
+id: 5A435F05-412D-4FC9-930A-0E6295760A78
+name: "Vrouter Load Ballancer Alpine 3.5 - KVM"
```

Comment:
We have problems on the OpenNebula side right now, when the appliance name updates in the future. There is an "Alpine 3.5", which can be expected to update to correspond with the installed system version. I don't know how much you want to support the image in the future, I only guess you'll want. And I guess the system (version) is not that important here, it's the load balancing functionality.

So, can you please skip the OS version in appliance name and change it to e.g.:
Vrouter Load Balancer Alpine - KVM